### PR TITLE
Detecting persistence behavior using AdminSDholder

### PR DESCRIPTION
**Pull request by 12306Bro**
Detects persistence behavior using AdminSDholder and fixes previous bug issues

### CODE REVIEW

**Comment by 12306Bro** [comment]
ffda0bd0-b189-4a64-bc4c-73854e9b8f85

**Comment by 12306Bro** [comment]
There are rules with missing or malformed 'id' fields. Create an id (e.g. here: https://www.uuidgenerator.net/version4) and add it to the reported rule(s).

What should I do?

File: rules/windows/builtin/win_ad_ACL_AdminSDHolder.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: Windows-ACL-AdminSDHolder
+description: Detecting persistence behavior using AdminSDholder
```
Comment (12306Bro) [inline rules/windows/builtin/win_ad_ACL_AdminSDHolder.yml:2]: id: 618aca70-906b-11ea-bb37-0242ac130002

**Comment by 12306Bro** [comment]
update

**Comment by Neo23x0** [comment]
Timespan of 60 minutes?

**Comment by 12306Bro** [comment]
The SDPROP process runs every 60 minutes on the PDC emulator and re-marks the ACL with the security permissions set by AdminSDHolder. The trigger frequency of SDPROP is the range of minimum 1 minute and maximum 120 minutes, so the intermediate value of 60 minutes is selected here. During my actual test, 5136 and 4780 were triggered within 60 minutes.

发件人: Florian Roth <notifications@github.com>
发送时间: 2020年5月14日 19:32
收件人: Neo23x0/sigma <sigma@noreply.github.com>
抄送: 12306Bro <xinliu12306@outlook.com>; State change <state_change@noreply.github.com>
主题: Re: [Neo23x0/sigma] Detecting persistence behavior using AdminSDholder (#730)


Timespan of 60 minutes?

**Pull request closed without merging**